doc/ledger.1

```diff
@@ -845,7 +851,10 @@ but show both sides of each transaction.
 .\".It Fl \-revalued
 .\".It Fl \-revalued-only
 .\".It Fl \-revalued-total Ar EXPR
-.\".It Fl \-rich-data
+.It Fl \-rich-data
+When generating ledger transaction from CSV file using the
+.Ic convert
+command, add CSV, Imported, and UUID meta-data.
```

Comment:
Please add an explanation for each metadata, e.g.
* CSV: the original csv 
* Imported: Date when the cvs was imported into ledger
* UUID: a uuid (I think based on  a checksum of the original csv) uniquely identifying this transaction